deploy/example/snapshot/README.md

```diff
@@ -0,0 +1,39 @@
+# Snapshot Example
+
+## Create a Source PVC
+
+```console
+kubectl apply -f $GOPATH/src/github.com/kubernetes-sigs/azuredisk-csi-driver/deploy/example/storageclass-azuredisk-csi.yaml
+kubectl apply -f $GOPATH/src/github.com/kubernetes-sigs/azuredisk-csi-driver/deploy/example/pvc-azuredisk-csi.yaml
+kubectl apply -f $GOPATH/src/github.com/kubernetes-sigs/azuredisk-csi-driver/deploy/example/nginx-pod-azuredisk.yaml
+```
+
+### Check the Source PVC
+
+```console
+$ kubectl exec nginx-azuredisk -- ls /mnt/azuredisk
+lost+found
+outfile
+```
+
+## Create a Snapshot of the Source PVC
+
+```console
+kubectl apply -f $GOPATH/src/github.com/kubernetes-sigs/azuredisk-csi-driver/deploy/example/snapshot/storageclass-azuredisk-snapshot.yaml
+kubectl apply -f $GOPATH/src/github.com/kubernetes-sigs/azuredisk-csi-driver/deploy/example/snapshot/azuredisk-volume-snapshot.yaml
+```
+
```

Comment:
also add some statements on how to check the creation or restore status?

Reply:
done.